stdlib/Dates/src/types.jl

```diff
@@ -83,6 +83,7 @@ abstract type Calendar <: AbstractTime end
 # ISOCalendar provides interpretation rules for UTInstants to civil date and time parts
 struct ISOCalendar <: Calendar end
 
+#Time Zones in ISO 8601 are represented as an offset from UTC
```

Comment:
I don't think we really need this comment as Dates doesn't really do anything timezone-related.